lib/core/include/apiHandler.hpp

```diff
@@ -78,11 +77,15 @@ namespace irods {
         );
     }
 
-    template < typename... types_t >
+    template <typename... types_t>
     class api_call_adaptor {
+    private:
         std::function<int(rsComm_t*, types_t...)> fcn_;
+
     public:
-        api_call_adaptor( std::function<int(rsComm_t*, types_t...)> _fcn ): fcn_(_fcn) {
+        api_call_adaptor( std::function<int(rsComm_t*, types_t...)> _fcn )
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Class 'api_call_adaptor' has a constructor with 1 argument that is not explicit.](https://app.codacy.com/manual/trel/irods/pullRequest?prid=5139965)